input/data/2016/2016arXiv161003751S/tev-000030-1-sed.ecsv

```diff
@@ -0,0 +1,26 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_ref, datatype: float32}
+# - {name: e_lo, datatype: float32}
```

Comment:
Looking at the values below, I'm pretty sure the columns should be `e_min` and `e_max` instead of `e_lo` and `e_hi` as you put here in the SED
See spec: http://gamma-astro-data-formats.readthedocs.io/en/latest/results/flux_points/index.html#energy-columns
and here for how we process energy columns:
https://github.com/gammapy/gamma-cat/blob/master/gammacat/sed.py#L67

I think this needs to be fixed in all (or at least several) SED files.